e2e/cypress/integration/channel_sidebar/sidebar_channel_menu_spec.js

```diff
@@ -0,0 +1,161 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+
+// ***************************************************************
+// - [#] indicates a test step (e.g. # Go to a page)
+// - [*] indicates an assertion (e.g. * Check the title)
+// - Use element ID when selecting an element. Create one if none.
+// ***************************************************************
+
+// Stage: @prod
+// Group: @channel_sidebar
+
+import {
+    beMuted,
+    beRead,
+    beUnmuted,
+    beUnread,
+} from '../../support/assertions';
+import {getAdminAccount} from '../../support/env';
+
+import {getRandomId, stubClipboard} from '../../utils';
+
+describe('Sidebar channel menu', () => {
+    const sysadmin = getAdminAccount();
+
+    let teamName;
+
+    before(() => {
+        cy.apiUpdateConfig({
+            ServiceSettings: {
+                ExperimentalChannelSidebarOrganization: 'default_on',
+            },
+        });
+
+        cy.apiInitSetup({loginAfter: true}).then(({team}) => {
+            teamName = team.name;
+
+            cy.visit(`/${team.name}/channels/town-square`);
+        });
+    });
+
+    it('should be able to mark a channel as read', () => {
+        // # Start in Town Square
+        cy.get('#sidebarItem_town-square').click();
+        cy.get('#channelHeaderTitle').should('contain', 'Town Square');
+
+        // # Save the ID of the Town Square channel for later
+        cy.getCurrentChannelId().as('townSquareId');
+
+        // # Switch to the Off Topic channel
+        cy.get('#sidebarItem_off-topic').click();
+        cy.get('#channelHeaderTitle').should('contain', 'Off-Topic');
+
+        // # Have another user send a message in the Town Square
+        cy.get('@townSquareId').then((channelId) => {
+            cy.postMessageAs({
+                sender: sysadmin,
+                message: 'post1',
+                channelId,
+            });
+        });
+
+        // * Verify that the Town Square channel is now unread
+        cy.get('#sidebarItem_town-square').should(beUnread);
+
+        // # Open the channel menu and select the Mark as Read option
+        cy.get('#sidebarItem_town-square').find('.SidebarMenu_menuButton').click({force: true});
+        cy.get('.SidebarMenu').contains('.MenuItem', 'Mark as Read').click();
+
+        // * Verify that the Town Square channel is now unread
```

Comment:
Small typo - update comment text to `* Verify that the Town Square channel is now read`